src/ajax.js

```diff
@@ -13,6 +13,7 @@ define([
 
 var
 	rhash = /#.*$/,
+	rextraQuestions = /\?+/,
```

Comment:
Ahhhh I understood :)

It's really different with that behaviour :/
Because if we want to be friendly on `.html??var=` we have to be also friendly on `/callback=???var=` and `/callback=?/??var=` the regex is more complex and maybe it's not so necessary.

I think we have to use this regex: `.replace( /([^=]|=\?)\?+/, "$1?" )`
(some tests here: http://jsfiddle.net/Mr21/0sofoaay/)